test/FastTests/Corax/CoraxPhraseQueries.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System.Linq;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Indexes;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.Corax;
+
+public class CoraxPhraseQueries : RavenTestBase
+{
+    public CoraxPhraseQueries(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [RavenTheory(RavenTestCategory.Corax | RavenTestCategory.Querying)]
+    [RavenData(SearchEngineMode = RavenSearchEngineMode.Corax)]
+    public void CanIndexWithMoreThan128Terms(Options options)
+    {
+        using var store = GetDocumentStore(options);
+        using var session = store.OpenSession();
+        session.Store(new Item() {FtsField = string.Join(" ", Enumerable.Range(0, 256).Select(x => $"unique{x}"))});
+        session.Store(new Item() {FtsField = string.Join(" ", Enumerable.Range(0, 256).Select(x => $"unique{256 - x}"))});
+
+        session.SaveChanges();
+        var results = session.Query<Item>().Customize(x => x.WaitForNonStaleResults()).Search(x => x.FtsField, "\"unique10 unique11\"").ToList();
+        Assert.Equal(1, results.Count);
+    }
+
+
+    [RavenTheory(RavenTestCategory.Corax | RavenTestCategory.Querying)]
```

Comment:
Please remove `RavenTestCategory.Corax` since this is not Corax specific test. You provide `RavenSearchEngineMode.All` below